test/k8s-local-cluster-test/run-test

```diff
@@ -11,10 +11,12 @@ PROVISION_CLUSTER_ARGS=""
 DELETE_CLUSTER_ARGS=""
 DEFAULT_NODE_TERMINATION_HANDLER_DOCKER_IMG="node-termination-handler:customtest"
 NODE_TERMINATION_HANDLER_DOCKER_IMG=""
-DEFAULT_EC2_METADATA_DOCKER_IMG="ec2-metadata-test-proxy:customtest"
-EC2_METADATA_DOCKER_IMG=""
+DEFAULT_WEBHOOK_DOCKER_IMG="webhook-test-proxy:customtest"
+WEBHOOK_DOCKER_IMG=""
 OVERRIDE_PATH=0
 K8S_VERSION="1.16"
+AEMM_URL="amazon-ec2-metadata-mock-service.default.svc.cluster.local"
```

Comment:
Can you also export the AEMM download URL so that we don't have to repeat that long string in all the e2e tests?